soundata/datasets/mavd.py

```diff
@@ -0,0 +1,303 @@
+    ),  
+    "audio_test": download_utils.RemoteFileMetadata(
+        filename="audio_test.zip",
+        url="https://zenodo.org/record/3338727/files/audio_test.zip?download=1",
+        checksum="824b035fa76dc83919e312105db4214f"
+    ),
+    "audio_train": download_utils.RemoteFileMetadata(
+        filename="audio_train.zip",
+        url="https://zenodo.org/record/3338727/files/audio_train.zip?download=1",
+        checksum="c7a897623364f80c8ff3775d8dd601a2"
+    ),
+    "audio_validate": download_utils.RemoteFileMetadata(
+        filename="audio_validate.zip",
+        url="https://zenodo.org/record/3338727/files/audio_validate.zip?download=1",
+        checksum="d220cef03f99b2d108d708421240f86d"
+    )   
+}
+
+LICENSE_INFO = "Creative Commons Attribution 4.0 International"
+
+
+class Clip(core.Clip):
+    """MAVD-traffic Clip class
+
+    Args:
+        clip_id (str): id of the clip
+
+    Attributes:
+        events (soundata.annotation.Events): sound events with start time, end time, label and confidence.
+        audio_path (str): path to the audio file
+        split (str): subset the clip belongs to (for experiments): train, validate, or test.
+        clip_id (str): clip id
+
+    """
+
+    def __init__(
+        self,
+        clip_id,
+        data_home,
+        dataset_name,
+        index,
+        metadata,
+    ):
+        super().__init__(
+            clip_id,
+            data_home,
+            dataset_name,
+            index,
+            metadata,
+        )
+
+        self.audio_path = self.get_path("audio")
+        self.txt_path = self.get_path("txt")
+
+    @property
+    def audio(self) -> Optional[Tuple[np.ndarray, float]]:
+        """The clip's audio
+
+        Returns:
+            * np.ndarray - audio signal
+            * float - sample rate
+
+        """
+        return load_audio(self.audio_path)
+
+    @property
+    def split(self):
+        return self._clip_metadata.get("split")
+
+    @core.cached_property
+    def events(self) -> Optional[annotations.Events]:
+        return load_events(self.csv_path)
+
+@io.coerce_to_bytes_io
+def load_audio(fhandle: BinaryIO, sr=None) -> Tuple[np.ndarray, float]:
+    """Load a MAVD-traffic audio file.
+
+    Args:
+        fhandle (str or file-like): File-like object or path to audio file
+        sr (int or None): sample rate for loaded audio, None by default, which
+            uses the file's original sample rate of 44100 without resampling.
+
+    Returns:
+        * np.ndarray - the mono audio signal
+        * float - The sample rate of the audio file
+
+    """
+    audio, sr = librosa.load(fhandle, sr=sr, mono=True)
+    return audio, sr
+
+
+@io.coerce_to_string_io
+def load_events(fhandle: TextIO) -> annotations.Events:
+    """Load an MAVD-traffic sound events annotation file
+    Args:
+        fhandle (str or file-like): File-like object or path to the sound events annotation file
+    Raises:
+        IOError: if txt_path doesn't exist
```

Comment:
This is a nice to have but we don't have it anywhere else in the other loaders, maybe remove for consistency since the error its self-explained? What do you think?

Reply:
Yes, I agree. Removed.